Index: sites/all/modules/contrib/feeds/mappers/text.inc
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- sites/all/modules/contrib/feeds/mappers/text.inc	(date 1370440717000)
+++ sites/all/modules/contrib/feeds/mappers/text.inc	(revision )
@@ -35,6 +35,8 @@
  */
 function text_feeds_set_target($source, $entity, $target, $value) {
   if (empty($value)) {
+    // Be sure to empty fields in which the new value is empty.
+    $entity->$target = array('und' => array());
     return;
   }
 
